notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<p>Use sentence case in title</p><p> </p><p>Vertex AI model registry -&gt; Vertex AI Model Registry</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>